sw/airborne/subsystems/radio_control/sbus_common.c

```diff
@@ -54,6 +54,124 @@
 #endif
 
 
+
+/*
+ * S.bus decoder matrix. (extracted from PX4)
```

Comment:
I tried to implement [this](https://www.google.nl/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&ved=0ahUKEwjq2r_JnKHSAhWKfhoKHQQ9Aq8QFggjMAE&url=http%3A%2F%2Fforum.autoquad.org%2Fdownload%2Ffile.php%3Fid%3D2319%26sid%3Db9bf8e54c8231cfb51691db4a3733673&usg=AFQjCNG4SJJZK_DtNhTvczfZfYZVOi5eKg&sig2=g37S5wm92t7mXeJK3YVMUg), but then I found the working implementation of PX4 / Ardupilot. From what I understand on fora it seems to work with all known sbus stuff, and it work for me at least.

The main thing is that the end byte is different between the different sbus receivers. For original sbus it's just 0x0, for later generations it varies (e.g. per 4 blocks). In this PR, the original 0x0 end byte is accepted like normal, so it should just work. I just miss the equipment to test it. 
SBUS2 is extra different in the sense that it provides telemetry, but I have not had the time to implement that yet fully (but it is on the todo list). The foundation is there now.